src/Utils/TypeStatus.js

```diff
@@ -0,0 +1,4 @@
+export default [
```

Comment:
it's not an util, additionally it's not smth common as I understand

Reply:
I moved typeStatus into the TypeSettings folder